### PR TITLE
Emergency Lingerie Repair Pull

### PR DESCRIPTION
**Pull request by Incoming5643**
The new "neko underwear" performed an unspoken sin of lingerie design and had pixels for the underwear that were off the side of the character model.

This meant that these things can and do clip through a lot of outfits and look terrible. I've salvaged the cats as best as I can muster.

d10_icon

### CODE REVIEW

**Comment by francinum** [comment]
:+1:

**Comment by freerealestate** [comment]
goddamn that is ugly

**Comment by Chocobro** [comment]
@Incoming5643 there are tassles on the reference material. i only sprited them on the south-facing, and francinum told me about it a few days after it was merged so i was not interested in booting up my computer again to delete four pixels and re-pr
tl;dr thx 2 u
